cli_test/test_helpers.go

```diff
@@ -1467,65 +1464,6 @@ func newValidator(f *Fixtures, cfg testnet.Config, appCfg *srvconfig.Config, ctx
 	}
 }
 
-func (f *Fixtures) TxStoreWasm(wasmFilePath string, flags ...string) (testutil.BufferWriter, error) {
```

Comment:
How about comment out instead of delete?(Same for other deleted tests)

Reply:
Some have been commented-out in YML, but this change not only removes a series of code blocks but also includes a small parameter modification. Ultimately, it's necessary to revert back to the original WASM and IBC functionality by referring to and reviewing the differences carefully in this PR.